src/Relation/Binary/Morphism/Construct/On.agda

```diff
@@ -0,0 +1,35 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Construct IsRelHomomorphisms from a relation and a function
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Relation.Binary.Core using (Rel)
+
+module Relation.Binary.Morphism.Construct.On
```

Comment:
I was expecting a construction of something called `On`. Instead, what is really built is a `IsRelHomomorphism`. Or maybe it is the relation `_≈_` that is being 'built'? (The code and the comment and the name don't quite agree with each other.)

Reply:
I agree with @jamesmckinna that `On` is the consistent name. I also think that it should go here rather than under `Relation.Binary.Construct.On`